hyperjumptech/monika
# Update Axios

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Implements #1007
2. Updates Axios to a newer version

## How to test
1. Using the test config below: 

```yaml
probes:
  - id: '1'
    name: 'status-200-test'
    requests:
      - url: https://httpbin.org/status/200
    alerts:
      - assertion: response.status < 200 or response.status > 299
        message: HTTP Status is not 200
      - assertion: response.time > 2000
        message: Response time is more than 2000ms
    incidentThreshold: 3
    recoveryThreshold: 3

  - id: '2'
    name: 'status-401-test'
    requests:
      - url: https://httpbin.org/status/401
    alerts:
      - assertion: response.status != 401
        message: HTTP Status is not 200
      - assertion: response.time > 2000
        message: Response time is more than 2000ms
    incidentThreshold: 3
    recoveryThreshold: 3

notifications:
  - id: desktop
    type: desktop
```
2. Run with `bin/run-dev -c test.yaml -s -1`
3. Expect everything to run without error

### monika mode probing
![image](https://user-images.githubusercontent.com/964106/225598790-24bc29f1-d453-43a8-ac48-00b3ae35293d.png)

### symon mode
![image](https://user-images.githubusercontent.com/964106/226271543-3594d88a-e319-46f4-bfa6-e562a639e418.png)
